### DISCUSSION TITLE
thanhle.blog/dau-nam-noi-chuyen-buong-bo

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#88] [Ideas]
# thanhle.blog/dau-nam-noi-chuyen-buong-bo

Chuyện gọt dũa bản thân

https://thanhle.blog/blog/dau-nam-noi-chuyen-buong-bo

**Comment by thanhlmm** [comment]
Đại văn hào Lev Tolstoy đã từng có câu nói: “Người ta như một phân số, mà tử số là giá trị thật của người ta, còn mẫu số là giá trị mà người ta tưởng tượng là mình có. Mẫu số càng to thì phân số càng nhỏ. Khi mẫu số là vô cùng tận thì phân số bằng 0”

À nói về chuyện cái "tôi" một xíu. Mình nghĩ cái tôi không phải là thứ mà ai cũng bảo phải bỏ bớt, mà là nuôi dưỡng đúng cách

**Reply by thanhlmm** [reply]
![CleanShot 2024-01-04 at 09 26 29@2x](https://github.com/thanhlmm/blog/assets/9281080/7242bdb3-1901-4da3-93ea-8e4f4f7c985b)
Khi thấy bản thân quá ngooo, hãy đọc sách

**Comment by chungnq253** [comment]
Ngộ tính cùng cách diễn đạt bài này thực sự hay. Chúc Thành năm mới rót thêm nhiều trà ngon và bỏ bớt đi những bã chè 🚀

**Comment by khiem20tc** [comment]
Năm rồi em cũng mất nhiều thứ chắc là để bản thân có chỗ trống dung nạp thêm điều mới. Bài viết anh Thành dạo này càng chín và có gì đó tĩnh hơn dạo trước. Chúc anh năm mới bình an :heart:

**Comment by thanhlmm** [comment]
https://www.youtube.com/watch?v=-vQLNy5hnfU&ab_channel=HoangNguyen

Chuyên buông bỏ của một người anh mới ra 2 ngày trước, dưới một góc nhìn khác sâu sắc hơn

**Comment by hai123vn** [comment]
dè sơ, năm mới chúc anh thật nhiều sức khỏe cho các dự án tiếp theo mà anh theo đuổi. Cảm ơn anh đã viết ra 1 blog thật là chất lượng cho em có thêm nhiều góc nhìn về cuộc sống hơn 💖

**Comment by nguyenhoaikhang37** [comment]
Trong khi nhà nhà ra hàng tá content như "nên lên kế hoạch gì cho năm tới", "năm 2024 bạn cần đặt mục tiêu gì để hướng tới thành công" thì em được 1 cú vã thẳng vào mặt =))) Nên "buông" gì? Vài ngày gần đây em cứ chạy theo số đông là cứ đổ vào chén trà của mình mà k nhìn lại là nó đã đầy cmnr T.T 

Qua 1 bài blog cứ ngỡ đọc xong cuốn sách self help dạng minimalism :)) nhma nó đời hơn, dễ thấm hơn 

Lại 1 bài viết  giúp em có thêm nhiều góc nhìn thú vị nữa <3 Chúc bình trà của anh Thành trong năm mới Hình - Sắc - Hương - Vị - Hóa đều xôm tụ và sớm trở thành CEO triệu đô^^
